### PR TITLE
Added LensDirt Texture Saving

### LINKED ISSUE #6: Change Bloom LensDirt Texture Parameter Save Friendly
I need path to texture to embed bloom info to the scene.
but current bloom LensDirt only stores Texture values. 🤔

### PR DESCRIPTION
**Pull request by hooh-hooah**
**Notable changes:**
Changed PostProcessingManager
- DirtLists are no longer get  instanced.
- Saves path of Dirt Texture, Try to find index of dirt texture path from manager when loading the preset.

This Merge Request will close #6

### CODE REVIEW

**Comment by hooh-hooah** [comment]
This issue will close #6

File: AI_Graphics/AI_Graphics/Inspector/Inspector.cs [hunk 1]
```diff
@@ -1,8 +1,7 @@
 ﻿using UnityEngine;
 using static AIGraphics.Inspector.Util;
 
-namespace AIGraphics.Inspector
-{
+namespace AIGraphics.Inspector {
```
Comment (ore-) [inline AI_Graphics/AI_Graphics/Inspector/Inspector.cs:4]: I guess this is unintentional? could we revert?

File: AI_Graphics/AI_Graphics/PostProcessingManager.cs [hunk 2]
```diff
@@ -9,9 +9,19 @@ namespace AIGraphics
     internal class PostProcessingManager : MonoBehaviour
     {
         private string _lensDirtTexturePath;
-        internal List<string> LensDirtPaths { get; private set; }
+        // Texture Lists don't have to be instanced.
+        internal static List<string> LensDirtPaths { get; private set; }
+        internal static List<Texture2D> LensDirts { get; private set; }
         internal List<Texture2D> LensDirtPreviews { get; private set; }
```
Comment (ore-) [inline AI_Graphics/AI_Graphics/PostProcessingManager.cs:15]: any reason why previews are left as instanced?

File: AI_Graphics/AI_Graphics/Setting/PostProcessingSettings.cs [hunk 3]
```diff
@@ -163,6 +163,10 @@ public enum Antialiasing {
         internal Transform VolumeTriggerSetting {
             get => _postProcessLayer.volumeTrigger;
         }
+        
+        internal string BloomTexturePath {
```
Comment (ore-) [inline AI_Graphics/AI_Graphics/Setting/PostProcessingSettings.cs:167]: what uses BloomTexturePath?

File: AI_Graphics/AI_Graphics/Inspector/Inspector.cs (hunk 1 above)
Comment (hooh-hooah) [reply AI_Graphics/AI_Graphics/Inspector/Inspector.cs:4]: Well my Visual Studio Sometimes does that idk why

File: AI_Graphics/AI_Graphics/PostProcessingManager.cs (hunk 2 above)
Comment (hooh-hooah) [reply AI_Graphics/AI_Graphics/PostProcessingManager.cs:15]: Well I think I forgot to make it static

File: AI_Graphics/AI_Graphics/Setting/PostProcessingSettings.cs (hunk 3 above)
Comment (hooh-hooah) [reply AI_Graphics/AI_Graphics/Setting/PostProcessingSettings.cs:167]: It will set bloom parameter's texture path value without exposing paramBloom.

Comment (ore-) [reply AI_Graphics/AI_Graphics/Setting/PostProcessingSettings.cs:167]: Just don't see it used anywhere in the commit. If you already have subsequent change that uses it, feel free to resolve the conversation, if not, please remove.

File: AI_Graphics/AI_Graphics/PostProcessingManager.cs (hunk 2 above)
Comment (ore-) [reply AI_Graphics/AI_Graphics/PostProcessingManager.cs:15]: if you plan to change it in a subsequent change, please resolve convo, then I'll approve so we can merge this, thx

**Comment by hooh-hooah** [comment]
> * Only things that need addressing are BloomTexturePath and LensDirtPreviews, rest looks fine, thanks.

Fixed with 2 commits.

**Pull request merged**